src/hamlib.mk

```diff
@@ -6,14 +6,18 @@ PKG             := hamlib
 $(PKG)_WEBSITE  := http://www.hamlib.org/
 $(PKG)_DESCR    := HamLib
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 3.1
-$(PKG)_CHECKSUM := 682304c3e88ff6ccfd6a5fc28b33bcc95d2d0a54321973fef015ff62570c994e
+$(PKG)_VERSION  := 3.2
+$(PKG)_CHECKSUM := b55cb5e6a8e876cceb86590c594ea5a6eb5dff2e30fc13ce053b46baa6d7ad1d
 $(PKG)_GH_CONF  := hamlib/hamlib/releases/latest
 $(PKG)_DEPS     := cc libltdl libusb1 libxml2 pthreads
 
+#LIBS='-lusb-1.0' \
+
 define $(PKG)_BUILD
+
     cd '$(BUILD_DIR)' && '$(SOURCE_DIR)/configure' \
         $(MXE_CONFIGURE_OPTS) \
+        LIBS='-lusb-1.0' \
```

Comment:
This only seems to be required for `i686-w64-mingw32.static` - there's some subtle failure in the detection for this target that is probably worth investigating.